packages/web-components/src/components/va-file-input/va-file-input.scss

```diff
@@ -137,13 +137,13 @@
   }
 
   .file-info-section {
-    border-bottom: 1px solid var(--vads-color-base-lighter);
     padding: 0 8px 8px;
     display: flex;
     align-items: center;
   }
 
   .file-button-section {
+    margin-top: 16px;
```

Comment:
@Andrew565 Let's make it 8px instead of 16px, please. Otherwise, all looks good.